ci/gpu/build.sh

```diff
@@ -55,6 +55,15 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "rapids-notebook-env=$MINOR_VERSION.*" \
       "rapids-doc-env=$MINOR_VERSION.*"
 
+
+export CUDA_PATH=/usr/local/cuda
+pip install cupy==8.0.0b5
+
+conda install -y -c numba \
+    numba \
+    llvmlite
```

Comment:
these are part of the rapids-build-env already

Reply:
There's a problem with the package of the `conda-forge` channel, it needs to be downloaded from `numba` channel. I removed this lines and updated the environment files instead.